react/features/base/tracks/middleware.web.ts

```diff
@@ -108,15 +105,9 @@ MiddlewareRegistry.register(store => next => action => {
         const isVideoTrack = jitsiTrack.type !== MEDIA_TYPE.AUDIO;
 
         if (isVideoTrack) {
-            // Do not change the video mute state for local presenter tracks.
-            if (jitsiTrack.type === MEDIA_TYPE.PRESENTER) {
-                APP.conference.mutePresenter(muted);
-            } else if (jitsiTrack.isLocal() && !(jitsiTrack.getVideoType() === VIDEO_TYPE.DESKTOP)) {
+            if (jitsiTrack.isLocal() && !(jitsiTrack.getVideoType() === VIDEO_TYPE.DESKTOP)) {
                 APP.conference.setVideoMuteStatus();
-            } else if (jitsiTrack.isLocal() && muted && jitsiTrack.getVideoType() === VIDEO_TYPE.DESKTOP) {
-                !getMultipleVideoSendingSupportFeatureFlag(state)
-                    && store.dispatch(toggleScreensharing(false, false, true));
-            } else {
+            } else if (!jitsiTrack.isLocal() && muted) {
```

Comment:
I don't know if this is possible but before if we have had local desktop track with `muted === false` we were executing `APP.UI.setVideoMuted(participantID);` and now not. I think we have the same thing for all remote tracks no matter of the `muted ` value. Shouldn't you use `||` here?

Reply:
We were executing `APP.UI.setVideoMuted(participantID)` in the legacy SS mode whenever SS starts. That is because we were automatically muting the camera when sharing starts which is not the case now. You are right, `APP.UI.setVideoMuted(participantID) `needs to be called for all remote tracks irrespective of the mute state. I will fix that.